### PR TITLE
US12789: read time

### PR DESCRIPTION
**Pull request by lukemoderwell**
PR adds:
- read time calculated by taking word count and dividing by 184 (avg words per min read)
- include for read time markup
- abstract article card to an include

### CODE REVIEW

File: _includes/_article-card.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div class="card card--layered">
+  <a href="{{ post.url }}">
+    <div class="card-bgImage sixteen-nine" style="background-image: url('{{ post.featured_image }}');"></div>
+    <div class="card-block">
+        <h5 class="publication_meta metadata flush">
+        {% assign words = post.content | number_of_words %}
```
Comment (tcmacdonald) [inline _includes/_article-card.html]: Can this be refactored to use the `read-time.html` partial?

File: _includes/_article-body.html [hunk 2]
```diff
@@ -1,9 +1,7 @@
 <article class="publication_body article font-family-base">
   <div class="publication_hero push-ends">
     <img src="{{ page.featured_image }}" alt="{{ page.featured_image.title }}" title="" style="width: 100%; height: 500px; object-fit: cover;" />
```
Comment (tcmacdonald) [inline _includes/_article-body.html]: Can we get rid of these styles?

File: _includes/_article-card.html (hunk 1 above)
Comment (lukemoderwell) [reply _includes/_article-card.html]: This was another case of `page` vs `post`. If you know a good way to programmatically switch between those, let me know.

**Comment by lukemoderwell** [comment]
@tcmacdonald made some changes to pass the content type in as a variable for the read time include. 
Changed the hero image to a background img to give us a little more control and eliminated the inline styles (other than the background-image). Also changed `publication` to `article` where it made sense. This was carried over from Maestro.

File: index.html [hunk 3]
```diff
@@ -18,24 +18,7 @@ <h1 class="section-header text-white">{{ page.heading }}</h1>
...
-            <p class="card-text">
-              <a href="{{ author.permalink }}">{{ post.author }}</a> |
-              <span class="card-date">
-                {{ post.date | date: date_format }}
-              </span>
-            </p>
-          </div>
-        </a>
-      </div>
+      {% include _article-card.html %}
```
Comment (tcmacdonald) [inline index.html]: And again, again.

**Pull request merged**